paper-collapse-item.html

```diff
@@ -71,6 +70,11 @@
 				@apply --paper-font-body1;
 				@apply --paper-collapse-item-content;
 			}
+			
+			.toogle {
+				position:absolute;
+				right:0px;
```

Comment:
Can you add a space after ':' to match the existing style?

Reply:
Sure, It's fixed.